packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -42,15 +48,42 @@ export async function getSupportedNodeVersion(
         );
       }
     } else {
-      if (!silent) {
-        throw new Error(
-          'Found `engines` in `package.json` with an unsupported node range: ' +
-            engineRange +
-            '\nPlease use one of the following supported ranges: ' +
-            JSON.stringify(supportedOptions.map(o => o.range))
-        );
-      }
+      throw new Error(
+        'Found `engines` in `package.json` with an unsupported Node.js version range: ' +
+          engineRange +
+          '\nPlease use one of the following supported ranges: ' +
+          JSON.stringify(supportedOptions.map(o => o.range)) +
+          '\nThis change is the result of a decision made by an upstream infrastructure provider (AWS).' +
```

Comment:
Won't this error be shown if I do i.e. `6.x` (which is a not-exiting runtime) or like `14.x` (version does not yet exist). Perhaps the "This changes is the result…" part should only be included if `isDiscontinued()` is true.

Reply:
@TooTallNate Nice catch. I updated the code to use `allOptions` and `supportedOptions`